packages/oui-checkbox/README.md

```diff
@@ -44,38 +44,32 @@
 
 ```html:preview
 <div ng-init="$ctrl.indeterminate = null">
-  <oui-checkbox text="Normal" description="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque euismod magna rutrum lectus gravida semper. Praesent ultrices feugiat enim, nec cursus ipsum fringilla luctus. Sed vel ornare neque. Vestibulum bibendum."></oui-checkbox>
-  <oui-checkbox text="Disabled" description="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque euismod magna rutrum lectus gravida semper. Praesent ultrices feugiat enim, nec cursus ipsum fringilla luctus. Sed vel ornare neque. Vestibulum bibendum." disabled></oui-checkbox>
+  <oui-checkbox description="Lorem ipsum dolor sit amet, consectetur adipiscing elit.">Normal</oui-checkbox>
+  <oui-checkbox description="Pellentesque euismod magna rutrum lectus gravida semper." disabled>Disabled</oui-checkbox>
 </div>
 ```
 
 ### On change
 
-**Note:** Model will not be refreshed until the `on-change` callback as not finished. If you want to access the new model inside the `on-change` callback you need to use the `modelValue` variable as below.
+**Note**: Model will not be refreshed until the `on-change` callback as not finished. If you want to access the new model inside the `on-change` callback you need to use the `modelValue` variable as below.
```

Comment:
`If you want to access the new model inside the ``on-change`` callback you need to use the `modelValue` variable as below.` 

->

`If you want to access the new model inside the ``on-change`` callback, you need to use the `modelValue` variable as below.`